pkg/command/ceip_participation_test.go

```diff
@@ -118,6 +118,10 @@ var _ = Describe("ceip-participation command tests", func() {
 })
 
 func TestCompletionCeip(t *testing.T) {
+	// This is global logic and needs not be tested for eac
```

Comment:
nit: Another places as well.
```suggestion
	// This is global logic and needs not be tested for each
```